data/config/adaguc.sld.xml

```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<Configuration>
+
+  <TempDir value="{ADAGUC_TMP}"/>
+  <Path value="{ADAGUC_PATH}/data"/>
+  <DataBase dbtype="sqlite" parameters="{ADAGUC_TMP}/adaguc.testdata.db"/>
+  
+  <!-- WMS settings -->
+  <WMS>
+    <Title>ADAGUC WMS Service</Title>
+    <Abstract>This service demonstrates how the ADAGUC server can be used to create OGC services.</Abstract>
+    <RootLayer>
+      <Title>Root Layer Title</Title>
+      <Abstract>Root Layer Abstract</Abstract>
+    </RootLayer>
+    <!-- PNG 8 bit requests are forced to 32 bit: better for antialiasing and combining layers with many colors -->
+<!--    <WMSFormat name="image/png" format="image/png32"/> -->
+  </WMS>
+  
+  
+  <!-- WMS -->
+  <Include location="{ADAGUC_PATH}/data/config/includes/WMS.include.xml"/>
+
+  <!-- WCS settings -->
+  <WCS>
+    <Title>ADAGUC WCS Service</Title>
+    <Label>wcsLabel</Label>
+  </WCS>
+
+  <!-- WCSFormat -->
+  <Include location="{ADAGUC_PATH}/data/config/includes/WCSFormat.include.xml"/>
+
+  <!-- Projections -->
+  <Include location="{ADAGUC_PATH}/data/config/includes/Projection.include.xml"/>
+           
+  <!--Legends -->
+  <Include location="{ADAGUC_PATH}/data/config/includes/DefaultStyles.include.xml"/>
+
+  <Include location="{ADAGUC_PATH}/data/config/includes/DefaultLayers.include.xml"/>
+  
+
+  <Layer type="grid">
+    <Group value="baselayers"/>
+    <Name force="true">grid10</Name>
+    <Title>grid 10 degrees</Title>
+    <Grid resolution="10"/>
+    <WMSFormat name="image/png32"/>
+  </Layer>
+
+  <!-- Layers -->
+  <Layer type="database">
+    <Name>radar</Name>
+    <FilePath filter="">/nobackup/users/bennekom/data/radardata.nc</FilePath>
+    <Variable>precipitation</Variable>
+  </Layer>
+
+
+  <Layer type="database">
+    <Name>depth</Name>
+    <FilePath filter="">/nobackup/users/bennekom/data/mv100.nc</FilePath>
```

Comment:
Maybe remove this layer and make this adaguc.sld.xml work with datafiles already in the adaguc-server repository: {ADAGUC_PATH}/data/datasets/testdata.nc